corehq/ex-submodules/soil/templates/soil/partials/dl_status.html

```diff
@@ -20,8 +20,7 @@ <h4>{% trans "There was an error processing your request" %}</h4>
         <p class="alert alert-info">
           <small>
             <i class="fa fa-info-circle"></i>&nbsp;{% blocktrans %}
-            If you are downloading a large file over a slow or unstable connection, we recommend syncing to Dropbox. Learn more about
-            <a href="https://confluence.dimagi.com/display/commcarepublic/Syncing+Downloads+to+Dropbox">Sync to Dropbox</a>.
+            If you are downloading a large file over a slow or unstable connection, we recommend syncing to Dropbox.
```

Comment:
The updated/broken links sheet calls this feature dead. Should this whole `{% if allow_dropbox_sync %}` section be removed? Maybe a question for @kcowger?